src/anet.c

```diff
@@ -381,16 +381,19 @@ static int anetSetReuseAddr(char *err, int fd) {
     return ANET_OK;
 }
 
-static int anetCreateSocket(char *err, int domain) {
+static int anetCreateSocket(char *err, int domain, int type, int protocol, int sockopts) {
     int s;
-    if ((s = socket(domain, SOCK_STREAM, 0)) == -1) {
+#ifdef SOCK_CLOEXEC
+    type |= SOCK_CLOEXEC;
+#endif
```

Comment:
let's add a comment that we don't currently rely on it, so we don't care if it isn't supported.

Reply:
Done.